### PR TITLE
test(e2e): add update components tests

### PR DESCRIPTION
**Pull request by innerdvations**
### What does it do?

adds tests for:
- adding an attribute to a component
- removing an attribute from a component
- deleting a component

also:
- updates playwright to 1.48.2
- adds unique suffix to playwright artifacts
- drops the hardcoded playwright versions from github workflows so it's not confusing to upgrade playwright in the future; from now on it will install whatever version is in package.json

### Why is it needed?

improve test coverage

### How to test it?

run the e2e tests

### Related issue(s)/PR(s)

parent pr: https://github.com/strapi/strapi/pull/21925

DX-1646

### CODE REVIEW

File: tests/e2e/tests/content-type-builder/components/update-components.spec.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+  test.afterAll(async () => {
+    await resetFiles();
+  });
+
+  test('Add attribute to component', async ({ page }) => {
+    await addAttributeToComponent(page, 'SomeComponent', addedAttribute);
+
+    // confirm that it exists in the content type this component was in
+    await navToHeader(page, ['Content-Type Builder', 'Mycollectiontype'], 'Mycollectiontype');
```
Comment (jhoward1994) [inline tests/e2e/tests/content-type-builder/components/update-components.spec.ts:92]: Maybe we could make these strings constants?

**Review by jhoward1994 — COMMENTED** [review]
The test looks good, once I merge develop locally and run `yarn playwright install` it is passing. Hopefully if we update the branch it'll pass here?

**Comment by innerdvations** [comment]
> The test looks good, once I merge develop locally and run `yarn playwright install` it is passing. Hopefully if we update the branch it'll pass here?

Yeah, I'll merge the parent branch first and then we'll see. If there's a problem I'll split off the playwright update to another PR

**Review by butcherZ — APPROVED** [review]
I ran all the tests, apart from `Remove attribute from component` being marked as flaky for firefox,  all LGTM!

File: tests/e2e/tests/content-releases/release-details-page.spec.ts [hunk 2]
```diff
@@ -56,9 +56,9 @@ describeOnCondition(edition === 'EE')('Release page', () => {
     await addEntryToRelease({ page, releaseName });
 
     // Publish the release
-    await page.getByRole('link', { name: 'Releases' }).click();
-    await page.getByRole('link', { name: `${releaseName}` }).click();
-    await page.getByRole('button', { name: 'Publish' }).click();
+    await clickAndWait(page, page.getByRole('link', { name: 'Releases' }));
+    await clickAndWait(page, page.getByRole('link', { name: `${releaseName}` }));
+    await clickAndWait(page, page.getByRole('button', { name: 'Publish', exact: true }));
```
Comment (innerdvations) [inline tests/e2e/tests/content-releases/release-details-page.spec.ts:61]: This test is failing on `develop` right now :( but I can just fix it here

**Review by Bassel17 — APPROVED** [review]
LGTM !

**Pull request merged**